test/debugger/helper-debugger-repl.js

```diff
@@ -34,11 +34,11 @@ let quit;
 
 function startDebugger(scriptToDebug) {
   scriptToDebug = process.env.NODE_DEBUGGER_TEST_SCRIPT ||
-                  common.fixturesDir + '/' + scriptToDebug;
+                  `${common.fixturesDir}/${scriptToDebug}`;
```

Comment:
Your call @vsemozhetbyt is you have the energy do it. I'll review it.
If you say not this PR I'll retract those comments.